[2024-12-05 03:17:41,487] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: crypto_data_pipeline.check_api_connection manual__2024-12-05T03:12:38.387350+00:00 [queued]>
[2024-12-05 03:17:41,493] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: crypto_data_pipeline.check_api_connection manual__2024-12-05T03:12:38.387350+00:00 [queued]>
[2024-12-05 03:17:41,494] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-12-05 03:17:41,494] {taskinstance.py:1357} INFO - Starting attempt 2 of 4
[2024-12-05 03:17:41,494] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-12-05 03:17:41,507] {taskinstance.py:1377} INFO - Executing <Task(HttpSensor): check_api_connection> on 2024-12-05 03:12:38.387350+00:00
[2024-12-05 03:17:41,509] {standard_task_runner.py:52} INFO - Started process 95 to run task
[2024-12-05 03:17:41,512] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'crypto_data_pipeline', 'check_api_connection', 'manual__2024-12-05T03:12:38.387350+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/crypto_data_pipeline.py', '--cfg-path', '/tmp/tmpzv3fxf3_', '--error-file', '/tmp/tmpst4zpdr0']
[2024-12-05 03:17:41,512] {standard_task_runner.py:80} INFO - Job 10: Subtask check_api_connection
[2024-12-05 03:17:41,553] {task_command.py:369} INFO - Running <TaskInstance: crypto_data_pipeline.check_api_connection manual__2024-12-05T03:12:38.387350+00:00 [running]> on host 85834e6b4d18
[2024-12-05 03:17:41,632] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=crypto_data_pipeline
AIRFLOW_CTX_TASK_ID=check_api_connection
AIRFLOW_CTX_EXECUTION_DATE=2024-12-05T03:12:38.387350+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-05T03:12:38.387350+00:00
[2024-12-05 03:17:41,633] {http.py:97} INFO - Poking: api/v3/coins/markets
[2024-12-05 03:17:41,646] {base.py:68} INFO - Using connection ID 'coin_gecko_api' for task execution.
[2024-12-05 03:17:41,648] {http.py:129} INFO - Sending 'GET' to url: https://api.coingecko.com/api/v3/coins/markets
[2024-12-05 03:17:42,147] {base.py:301} INFO - Success criteria met. Exiting.
[2024-12-05 03:17:42,162] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=crypto_data_pipeline, task_id=check_api_connection, execution_date=20241205T031238, start_date=20241205T031741, end_date=20241205T031742
[2024-12-05 03:17:42,205] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05 03:17:42,257] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
